config/crds/hive.openshift.io_hiveconfigs.yaml

```diff
@@ -1,22 +1,23 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   creationTimestamp: null
-  labels:
-    controller-tools.k8s.io: "1.0"
   name: hiveconfigs.hive.openshift.io
 spec:
   group: hive.openshift.io
   names:
     kind: HiveConfig
+    listKind: HiveConfigList
     plural: hiveconfigs
-  scope: Cluster
+    singular: hiveconfig
+  scope: ""
```

Comment:
The scope is messed up.

Reply:
This needs the same changes that I made in https://github.com/openshift/hive/pull/979.

Reply:
How would you like to proceed, combine both into one or merge 979 first and then rebase this.

Reply:
I pulled in the changes from #979 into this PR.